benchmark/spmv/spmv.cpp

```diff
@@ -96,6 +101,47 @@ void apply_spmv(const char *format_name, std::shared_ptr<gko::Executor> exec,
             system_matrix->apply(lend(b), lend(x_clone));
             exec->synchronize();
         }
+
+        // tuning run
+#ifdef GINKGO_BENCHMARK_ENABLE_TUNING
+        auto &format_case = spmv_case[format_name];
```

Comment:
add the check for coo and hybrid?

Reply:
This is supposed to be an example of tuning, and not specifically to tune that variable. The same scheme can be used to tune the SpMV formats in some other way, so I think a check for COO is not needed as if someone else tunes something else he will need the same code but maybe for CSR.

Reply:
I think it is more difficult to use it on CSR because csr is required srow built on the construction.
But I am fine with leaving it.